openassessment/xblock/static/sass/oa/views/_oa-base.scss

```diff
@@ -29,6 +29,56 @@
     padding: 0;
   }
 
+  // --------------------
+  // overall actions
+  // --------------------
+  .list--actions {
+    margin-bottom: ($baseline-v/2);
+    @include text-align(center);
+
+    @include media($bp-ds) {
+      @include text-align(right);
+    }
+
+    @include media($bp-dm) {
+      @include text-align(right);
+    }
+
+    @include media($bp-dl) {
+      @include text-align(right);
+    }
+
+    @include media($bp-dx) {
+      @include text-align(right);
+    }
+
+    // STATE: actions has an error
+    &.has--error {
+
+      .message {
+        margin-bottom: $baseline-v;
+        @include text-align(left);
+      }
+    }
+
+    .action--submit {
```

Comment:
I think this is the style you want on the buttons that are looking funny but they don't have the list--actions parent.